recipes/expat/expat.inc

```diff
@@ -6,6 +6,9 @@ RECIPE_TYPES = "machine native sdk"
 
 inherit autotools make-vpath
 
+#Only used for Aarch64
+AUTOCONFDIR = "/conftools"
```

Comment:
ok, but you could just only set that var for that arch, I would much prefer that anyway